package.json

```diff
@@ -15,5 +15,8 @@
   },
   "pre-commit": [
     "check"
-  ]
+  ],
+  "dependencies": {
+    "local-ssl-proxy": "^1.3.0"
+  }
```

Comment:
Should this be in dependencies?

Reply:
No that is just a local dependency just for testing